src/util/SchemaUtil.h

```diff
@@ -24,6 +24,9 @@ class SchemaUtil final {
     SchemaUtil() = delete;
 
 public:
+    // The length limitation of schema comment field.
+    static constexpr std::size_t kCommentLengthLimit = 256;
```

Comment:
Maybe we can allow a longer length limit, i feel `256` is not enough since it could not be configured.

Reply:
Change it to gflag?